lib/rules/camelcase.js

```diff
@@ -60,6 +79,18 @@ module.exports = {
             return name.indexOf("_") > -1 && name !== name.toUpperCase();
         }
 
+        /**
+         * Checks if a string match the ignore list
+         * @param {string} name The string to check.
+         * @returns {boolean} if the string is ignored
+         * @private
+         */
+        function isIgnored(name) {
+            return ignoreList.findIndex(
```

Comment:
It's better to use `!ignoreList.includes(/* */)`.

Reply:
The problem with includes is that I cannot execute a function to check values.

I need findIndex to check the ignoreList _(soon renamed to allowedList)_ regular expressions.